src/write_gcm.c

```diff
@@ -4406,6 +4472,7 @@ static int wg_init(void)
   wg_context_t *ctx = NULL;
   int result = -1;  // Pessimistically assume failure.
 
+  init_locks();
   curl_global_init(CURL_GLOBAL_SSL);
```

Comment:
[Optional] Funny, we never call `curl_global_cleanup`... Also, `curl_global_init` claims to do everything necessary to initialize the SSL layer, though apparently not...

Reply:
Probably a good idea, but I don't know what else will change if I do that. How about a follow-up?

Reply:
SG.